Afterpay payment method country available.

tiutiunnykleasoft opened the pull request:
Afterpay payment method country available. Rename IPBlockSubscriber.php -> PaymentKeeperSubscriber.php

tiutiunnykleasoft committed:
Afterpay payment method country available. Rename IPBlockSubscriber.php -> PaymentKeeperSubscriber.php

LeasoftKonstantinShapoval commented on custom/plugins/emspay/Resources/config.xml:
```diff
@@ -56,12 +64,16 @@
+            <label lang="nl">Landen beschikbaar voor AfterPay</label>
+            <label lang="fr">Pays disponibles pour AfterPay</label>
+            <value>NL, BE</value>
+            <description lang="en">To allow AfterPay to be used for any other country just add its country code (in ISO 2 standard) to the "Countries available for AfterPay" field. Example: BE, NL, FR.</description>
```
Needs add description for empty values:
EN "<br> If field is empty then AfterPay will be available for all countries."
DE "<br> Wenn das Feld leer ist, ist AfterPay fur alle Lander verfugbar."
FR "<br> Si le champ est vide, AfterPay sera disponible pour tous les pays."
NL "<br> Als het veld leeg is, is AfterPay beschikbaar voor alle landen."

tiutiunnykleasoft replied:
Thanks for localized answer,

tiutiunnykleasoft committed:
Add localization explaining for empty value of Afterpay country available

tiutiunnykleasoft committed:
Updated README.md

volgustleasoft commented on custom/plugins/emspay/Subscriber/PaymentKeeperSubscriber.php:
```diff
@@ -1,56 +1,74 @@
+     */
+
+    private function countryValidation($method,$country){
+        $countries_string = $this->config[implode('_',['emsonline',$method,'countries'])];
```
Could it be that we are accessing an array element by a key that does not exist? For example: $this->config['emsonline_ideal_countries']?

tiutiunnykleasoft replied:
there is all okay I have a check later

volgustleasoft commented on custom/plugins/emspay/Subscriber/PaymentKeeperSubscriber.php:
```diff
@@ -1,56 +1,74 @@
+     * @return bool
+     */
+    private function ipAddressValidation($method){
+        $test_ip = $this->config[implode('_',['emsonline_test_ip',$method])];
```
Could it be that we are accessing an array element by a key that does not exist? For example: $this->config['emsonline_test_ip_ideal'] ?

tiutiunnykleasoft replied:
there is all okay I have a check later

tiutiunnykleasoft committed:
Remove PaymentKeeperSubscriber.php
Reworked IDealIssuerSubscriber.php to be a ProcessPaymentDisplaySubscriber.php
Updated CHANGELOG.md and README.md to be a 1.1.0 version

tiutiunnykleasoft committed:
Merge branch 'development' into afterpay_countries

tiutiunnykleasoft committed:
Fix undefined array key notice in ProcessPaymentDisplaySubscriber.php

tiutiunnykleasoft committed:
Merge remote-tracking branch 'origin/afterpay_countries' into afterpay_countries
